app/assets/stylesheets/patternfly_overrides.scss

```diff
@@ -367,6 +367,10 @@ table.table.table-summary-screen tbody td img {
 /// begin quadicon styling
 /// ===================================================================
 
+.single-wrapper i {
+  color: #0099cc;
+}
```

Comment:
I have mixed feelings about this, if we want a global styling rule I think this should be living in the `ui-components` repo. Or if you don't want to touch that repo, I think the color should be explicitly specified in each quad decorator.

Reply:
@skateman I don't think this belongs in ui-components.  What's your reasoning for repeating this (34 times) in each  decorator?

Reply:
I know that repeating is bad...are you saying that this styling will be only used in the classic UI and nowhere else in the future, SUI maybe?

Reply:
Let's make it more safe then:
```scss
miq-quadicon > .single-wrapper {
  .fonticon > i {
    color: #0099cc;
  }
}
```

Reply:
@skateman I'm saying that ui-components is available for anyone to use, and we shouldn't lock-in any specific colors.

Reply:
ok